packages/vscode-extension/lib/metro_helpers.js

```diff
@@ -11,6 +11,34 @@ function requireFromAppDir(module, options) {
 
 function resolveFromAppDir(module, options) {
   const paths = options && options.paths ? [...options.paths, appRoot] : [appRoot];
+
+  return require.resolve(module, { paths });
+}
+
+function requireFromAppDependency(dependency, module, options) {
```

Comment:
VERY NIT: I'd change the order of parameters in the `FromAppDependency` function to have the `dependency` last -- it feels more natural to me that the signature is "like `require`" (so it takes `(module, options)`) + an extra parameter at the end.